src/adaptation/massmatrix.jl

```diff
@@ -102,20 +102,32 @@ function WelfordVar(sz::Union{Tuple{Int},Tuple{Int,Int}}; kwargs...)
     return WelfordVar{Float64}(sz; kwargs...)
 end
 
-function Base.resize!(wv::WelfordVar, θ::AbstractVecOrMat{T}) where {T<:AbstractFloat}
-    if size(θ) != size(wv.var)
+function resize_with!(wv::WelfordVar, θ::AbstractMatrix{T}) where {T<:AbstractFloat}
```

Comment:
`T` should be determined by `WelfordVar`, shouldn't it?
```suggestion
function resize_with!(wv::WelfordVar{T}, θ::AbstractMatrix{<:Real}) where {T<:AbstractFloat}
```
? Or possibly only matching inputs should be allowed?

Reply:
Now `T` is determined by `WelfordVar`